libs/area_threading.py

```diff
@@ -0,0 +1,42 @@
+import os
+from shutil import rmtree
+from threading import Thread
+from libs.distancing import Distancing as CvEngine
+from libs.area_reporting import AreaReporting as AreaEngine
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def run_area_processing(config, pipe, areas):
+    pid = os.getpid()
+    logger.info(f"[{pid}] taking on {len(areas)} cameras")
```

Comment:
This looks like copied from the other file. Could possibly be removed. Or the text updated